wp/sections/download.tex

```diff
@@ -1,8 +1,76 @@
+
+\begin{center}
+\texttt{\$ ./ktlint -R diktat.jar --disabled\_rules=standard "path/to/project/**/*.kt"}
+\end{center}
+
+After the run, all detected errors will displayed. Each warning contains of a rule name, a description of the rule and line/column where this error appears in the code. It also will contain \texttt{"cannot be auto-corrected"} note if the Inspection does not have autofixer. The format of warning is the following:
+
+\begin{center}
+/path/to/project/file.kt:6:5: [WARNING\_ID\_NAME] free text of the warning (cannot be auto-corrected)
+\end{center}
+
+Please also note, that as diktat is using ktlint framework - the format of the reported warnings can be changed: it can be xml, json and other formats that are supported by ktlint. Please refer to ktlint documentation \footnote{\url{https://github.com/pinterest/ktlint\#creating-a-reporter}} to see the information about custom reporters. 
+
+\subsection{Maven plugin}
+Maven plugin was introduced for diktat since the version 0.1.3. The following code snippet from \texttt{pom.xml} shows how to use diktat with Maven plugin:
+\begin{lstlisting}[caption={DiKTat with Maven plugin}, label={lst:maven}, language=Kotlin]
+          <plugin>
+                <groupId>org.cqfn.diktat</groupId>
+                <artifactId>diktat-maven-plugin</artifactId>
+                <version>${diktat.version}</version>
+                <executions>
+                    <execution>
+                        <id>diktat</id>
+                        <phase>none</phase>
+                        <goals>
+                            <goal>check</goal>
+                            <goal>fix</goal>
+                        </goals>
+                        <configuration>
+                            <inputs>
+                                <input>${project.basedir}/src/main/kotlin</input>
+                                <input>${project.basedir}/src/test/kotlin</input>
+                            </inputs>
+                            <diktatConfigFile>diktat-analysis.yml</diktatConfigFile>
+                           <excludes>
+                              <exclude>${project.basedir}/src/test/kotlin/excluded</exclude>
+                           </excludes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+\end{lstlisting}
+
+To run diktat in only-check mode use the command: \texttt{mvn diktat:check@diktat}. To run diktat in autocorrect mode use the command: \texttt{mvn diktat:fix@diktat}.
+
+
+\subsection{Gradle plugin}
+This plugin is available since version 0.1.5. The following code snippet shows how to configure Gradle plugin for diktat.
+
+\begin{lstlisting}[caption={DiKTat with Gradle plugin}, label={lst:gradle1}, language=Kotlin]
+plugins {
+    id("org.cqfn.diktat.diktat-gradle-plugin") version "0.1.7"
+}
+
+apply(plugin = "org.cqfn.diktat.diktat-gradle-plugin")
```

Comment:
```suggestion
```
This line is not needed if using plugin DSL, it's from the second option with buildscript.